modules/bouncer_scriptworker/manifests/settings.pp

```diff
@@ -57,34 +61,51 @@
+      'comm-thunderbird-dev' => {
+        worker_group             => 'tb-bouncer-dev',
+        worker_type              => 'bouncer-v1',
+        taskcluster_client_id    => 'project/comm/thunderbird/releng/scriptworker/bouncer/dev',
+        taskcluster_access_token => secret('comm_thunderbird_bouncer_scriptworker_taskcluster_access_token_dev'),
+        taskcluster_scope_prefix => 'project:comm:thunderbird:releng:bouncer:',
 
-    $script_config              = "${root}/script_config.json"
-    $script_config_content      = {
-        work_dir           => $work_dir,
-        schema_files       => {
-            submission => "${root}/lib/python${python3_virtualenv_version}/site-packages/bouncerscript/data/bouncer_submission_task_schema.json",
-            aliases    => "${root}/lib/python${python3_virtualenv_version}/site-packages/bouncerscript/data/bouncer_aliases_task_schema.json",
+        sign_chain_of_trust      => false,
+        verify_chain_of_trust    => true,
+        verify_cot_signature     => false,
+        cot_product              => 'thunderbird',
+        bouncer_instances        => {
+          "project:comm:thunderbird:releng:bouncer:server:staging" => {
+            api_root                  => 'https://admin-bouncer-releng.stage.mozaws.net/api',
+            timeout_in_seconds        => 60,
+            username                  => 'releng-tbird-staging',
+            password                  => secret('tbird-bouncer-staging_password'),
+          },
         },
-        verbose            => $verbose_logging,
-        bouncer_config     => $_env_config['bouncer_instances'],
+      },
+      'comm-thunderbird-prod' => {
+        worker_group             => 'tb-bouncer-v1',
+        worker_type              => 'bouncer-v1',
+        taskcluster_client_id    => 'project/comm/thunderbird/releng/scriptworker/bouncer/prod',
+        taskcluster_access_token => secret('comm_thunderbird_bouncer_scriptworker_taskcluster_access_token_prod'),
+        taskcluster_scope_prefix => 'project:comm:thunderbird:releng:bouncer:',
+        sign_chain_of_trust      => true,
+        verify_chain_of_trust    => true,
+        verify_cot_signature     => true,
+        cot_product              => 'thunderbird',
+        bouncer_instances        => {
+          "project:comm:thunderbird:releng:bouncer:server:staging" => {
+            api_root                  => 'https://admin-bouncer-releng.stage.mozaws.net/api',
+            timeout_in_seconds        => 60,
+            username                  => 'releng-tbird-staging',
+            password                  => secret('tbird-bouncer-staging_password'),
+          },
+          'project:comm:thunderbird:releng:bouncer:server:production' => {
+            api_root                  => 'https://bounceradmin.mozilla.com/api',
+            timeout_in_seconds        => 60,
+            username                  => 'ffxbld',
+            # TODO Split credentials
```

Comment:
Tiny nit: maybe reference a bug number if we have one filed? Totally fine if not.